crates/router/src/types/api.rs

```diff
@@ -172,41 +169,7 @@ pub enum ConnectorCallType {
     Single(ConnectorData),
 }
 
-#[derive(EnumString, Serialize, Deserialize, PartialEq, Debug)]
-#[strum(serialize_all = "lowercase")]
-enum ConnectorName {
-    Aci,
-    Adyen,
-    Airwallex,
-    Authorizedotnet,
-    Bambora,
-    Bluesnap,
-    Braintree,
-    Checkout,
-    Coinbase,
-    Cybersource,
-    Dlocal,
-    #[cfg(feature = "dummy_connector")]
-    DummyConnector,
-    Fiserv,
-    Forte,
-    Globalpay,
-    Klarna,
-    Mollie,
-    Nuvei,
-    Opennode,
-    Payu,
-    Rapyd,
-    Shift4,
-    Stripe,
-    Worldline,
-    Worldpay,
-    Multisafepay,
-    Nexinets,
-    Paypal,
-    Trustpay,
-    Zen,
-}
+use api_models::enums::Connector as ConnectorName;
```

Comment:
Can we avoid using this rename?

Reply:
There is a trait [link](https://github.com/juspay/hyperswitch/blob/c158e043fb4be0ad389ddc6baa3d7a82bb3065d9/crates/router/src/types/api.rs#LL102C21-L102C21) with the same name

Reply:
used `enums::Connector` because there's a trait named `Connector` already present in that file